### ISSUE TITLE
Feature request

### ISSUE THREAD

**Issue by mars01** [StamusNetworks/SELKS#58]
Hi,

Not sure if this is the appropriate place for these requests, but I'd like to see a future release of Scirius to allow exporting individual rule alert data. Under the Suricata color wheel in the SELKS WebUI, there is the list of triggered rules and the hits by host, source IP, and destination IP per each rule. It would be nice to be able to export the data from each individual rule with the IP address summaries to a format like CSV.

Thanks!

**Comment by pevma** [comment]
@mars01 - which dashboard and which visualization are you referring to? I think by default you can export csv from the visualization itself.

**Comment by mars01** [comment]
Hi Peter,

Thanks for the reply and great work with SELKS. In Scirius, under any of the Suricata rules where it provides metrics (Hits by host, Source IP, Destination IP, rule name, references, etc...) I'm not able to find an option to export those metrics to CSV. I see you can in Kibana, but not Scirius itself.

**Comment by pevma** [comment]
@mars01 - ok you mean Scirius I thought it was Kibana. Not yet in Scirius but it may be a good feature request @regit - what do you think ? should we move that request to Scirius?

**Comment by mars01** [comment]
Yep, Scirius would be helpful. I use the color wheel a lot for navigating
the Suricata alert categories because they best summarize affected IP
addresses in specific alerts. If there was some way to export that list of
information, that would definitely be helpful.

**Comment by pevma** [comment]
The Kibana dashboards can be adjusted in terms of display length and enlarged for that matter (if you click on ``edit`` in the top right corner of the Kibana dashboard and then edit the individual visualization to show more than 10/20/anything).
For Scirius -  would you please post a feature request on the Scirius github repo?
